src/WCSimPhysicsMessenger.cc

```diff
@@ -14,14 +14,14 @@ WCSimPhysicsMessenger::WCSimPhysicsMessenger(WCSimPhysicsList* WCSimPhys)
   WCSimDir->SetGuidance("Commands to change secondary interaction model for protons");
 
   hadmodelCmd = new G4UIcmdWithAString("/WCSim/physics/secondaries/model",this);
-  hadmodelCmd->SetGuidance("Available options: GHEISHA BERTINI BINARY");
+  hadmodelCmd->SetGuidance("Available options for low energy (< 10 GeV): BERTINI BINARY (default is BERTINI)");
```

Comment:
This was a suggestion from the G4developers, but we need to check if
the Bertini is better than Binary or vice-versa. The model isn't automatically
used it does need to be defined.